tests/tree_bulk_ingest.rs

```diff
@@ -107,16 +123,20 @@ fn blob_tree_bulk_ingest() -> lsm_tree::Result<()> {
     let tree = Config::new(folder, seqno.clone())
         .with_kv_separation(Some(KvSeparationOptions::default().separation_threshold(1)))
         .open()?;
-
-    tree.ingest(
-        (0..ITEM_COUNT as u64).map(|x| {
-            let k = x.to_be_bytes();
-            let v = nanoid::nanoid!();
-            (k.into(), v.into())
-        }),
-        &seqno,
-        &visible_seqno,
-    )?;
+    let tree = match tree {
+        AnyTree::Blob(t) => t,
+        _ => unreachable!(),
+    };
+
+    let seq = seqno.next();
+    let mut ingestion = BlobIngestion::new(&tree)?.with_seqno(seq);
```

Comment:
Because in fjall (and in general), the type of the tree is kind of opaque (AnyTree), it would be better for the ingestion API to just accept an AnyTree, and internally handle the difference (maybe using `dyn`).